CHANGES.md

```diff
@@ -10,6 +10,7 @@ Airbase 149
   - maven-source-plugin to 3.3.0 (from 3.0.1)
   - maven-surefire-plugin to 3.2.2 (from 3.2.1)
   - build-helper-plugin to 3.5.0 (from 3.4.0)
+  - maven-surefire-plugin 3.2.5 (from 3.2.2)
```

Comment:
Update the entry above instead of creating a new entry.

Reply:
ok

Reply:
done